common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -66,15 +70,26 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
       video.attr("layout", "responsive")
     }
 
+    def createFacebookVideoElement(document: Document, videoId: String, elementType: String): Element = {
+      val video = document.createElement(elementType)
+      video.attr("data-href", s"https://www.facebook.com/theguardian/videos/$videoId")
+      video.attr("data-embed-as", "video")
+      video.attr("width", "5")
+      video.attr("height", "3")
+      video.attr("layout", "responsive")
```

Comment:
Duplication of code with `createElement` which I think we should be able to combine.

Perhaps we should have something a bit like:

`sealed abstract class AmpExternalVideo(val videoId: String, val elementType: String, val customAttributes: Map[String, String])`

...and pass the custom attributes in the case class definition.  It might need to be a bit smarter than this because videoId is needed to define the `href`.  Simple (I haven't thought this through) solution would be to replace `Map[String, String]` with `String => Map[String, String]` function.